knative-operator/pkg/controller/knativeserving/servicemesh/servicemesh.go

```diff
@@ -239,6 +242,39 @@ func isServiceMeshMemberRollReady(servingNamespace string, api client.Client) (b
 	return false, nil
 }
 
+// create wide-open networkpolicies for the service mesh components
+func installNetworkPolicies(instanceNamespace string, api client.Client) error {
+	log.Info("Installing Mesh Network Policies")
+	namespace := ingressNamespace(instanceNamespace)
+	const path = "deploy/resources/serviceMesh/networkpolicies.yaml"
+
+	manifest, err := mf.NewManifest(path, false, api)
+	if err != nil {
+		log.Error(err, "Unable to create Mesh Network Policy install manifest")
+		return err
+	}
+	smcp := &maistrav1.ServiceMeshControlPlane{}
+	ready, err := isServiceMeshControlPlaneReady(instanceNamespace, api)
+	if err != nil {
+		return err
+	}
+	if ready {
```

Comment:
I guess this `ready` condition is not necessary. But if you want, you need to take care when `isServiceMeshControlPlaneReady()` returns `false, nil` from [here](https://github.com/openshift-knative/serverless-operator/pull/75/files#diff-4850e596b590230168aecc3c95fbd473R160).

Reply:
Sure thank you